pkgs/racket-test-core/tests/racket/for.rktl

```diff
@@ -16,16 +16,24 @@
                               (if 64-bit? (- (expt 2 60) 1) +536870911)
                               (if 64-bit? (- (expt 2 62) 1) +1073741823)))
 
+(define (five) 5)
+
 (test-sequence [(0 1 2)] 3)
 (test-sequence [(0 1 2)] (in-range 3))
 (test-sequence [(3 4 5)] (in-range 3 6))
 (test-sequence [(7 6 5)] (in-range 7 4 -1))
+(test-sequence [(5 6)] (in-range (five) 7))
+(test-sequence [(3 4)] (in-range 3 (five)))
+(test-sequence [(0 5)] (in-range 0 10 (five)))
 (test-sequence [(3.0 4.0 5.0)] (in-range 3.0 6.0))
 (test-sequence [(3.0 3.5 4.0 4.5 5.0 5.5)] (in-range 3.0 6.0 0.5))
 (test-sequence [(3.0 3.1 3.2)] (in-range 3.0 3.3 0.1))
 (test-sequence [(6 7)] (in-inclusive-range 6 7))
 (test-sequence [(3 4 5 6)] (in-inclusive-range 3 6))
 (test-sequence [(7 6 5 4)] (in-inclusive-range 7 4 -1))
+(test-sequence [(5 6 7)] (in-range (five) 7))
```

Comment:
I think you meant this to be `in-inclusive-range`, right?

Reply:
Fixed. Thanks!